functions.php

```diff
@@ -409,7 +409,7 @@ function wp_autoupdates_handle_themes_enable_disable() {
 		return;
 	}
 
-	// in core, the referer is setup in wp-admin/themes.php or wp-admin/network/themes.php.
+	// In core, the referrer is setup in wp-admin/themes.php or wp-admin/network/themes.php.
```

Comment:
While it's true that `referrer` is correct English usage, core specifically uses `referer` even in inline comments.  I'm not 100% sure about the history, but core's usage is mostly likely because of the misspelling in the original HTTP RFC (see https://en.wiktionary.org/wiki/referer#Usage_notes).

So, can you revise this PR to not "correct" that?